README.md

```diff
@@ -148,23 +148,21 @@ If you are trying to use the P2 plugin permanently, it likely makes more sense t
    ```
    And
    ```
-   + <feature name="nexus-repository-p2" description="org.sonatype.nexus.plugins:nexus-repository-p2" version="0.0.2">
+   + <feature name="nexus-repository-p2" description="org.sonatype.nexus.plugins:nexus-repository-p2" version="1.0.0">
    +     <details>org.sonatype.nexus.plugins:nexus-repository-p2</details>
-   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-p2/0.0.2</bundle>
+   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-p2/1.0.0</bundle>
    + </feature>
     </features>
    ```
 This will cause the plugin to be loaded and started with each startup of Nexus Repository.
 
 ## The Fine Print
-
-It is worth noting that this is **NOT SUPPORTED** by Sonatype, and is a contribution of ours
-to the open source community (read: you!)
+Starting from version 3.21+ the `p2` plugin is supported by Sonatype, but still is a contribution of ours to the open source community (read: you!)
 
 Remember:
 
 * Use this contribution at the risk tolerance that you have
-* Do NOT file Sonatype support tickets related to P2 support in regard to this plugin
+* Do NOT file Sonatype support tickets related to p2 support in regard to this plugin
 * DO file issues here on GitHub, so that the community can pitch in
```

Comment:
Is this still true?  I suspect we want JIRA (as mentioned above).

Reply:
@jlstephens89 what is the best solution? Should we keep community issues in github or we support them in Jira?

Reply:
I suspect we have to use jira like it is in the R readme https://github.com/sonatype-nexus-community/nexus-repository-r

Reply:
Done 018a1dddbd5a3ffd0dc188c53c6f337b1e861677